lambda/db/src/migrations/2022.02.24T11.35.00.add-additional-email-in-user-table.ts

```diff
@@ -0,0 +1,12 @@
+const { DataTypes } = require('sequelize');
+
+export const up = async ({ context: sequelize }) => {
+  await sequelize.getQueryInterface().addColumn('users', 'additional_email', {
+    type: DataTypes.STRING,
+    allowNull: true,
+  });
+};
+
+export const down = async ({ context: sequelize }) => {
```

Comment:
is the 'down' function so that we can roll back the migration? I'm curious about the context.

Reply:
`migrations` is a `time machine`, so that `down` function reverts the changes made for each migration script.